libraries/rush-lib/src/cli/actions/npm-check-typings.d.ts

```diff
@@ -0,0 +1,50 @@
+declare module NpmCheck {
```

Comment:
Are these typings for a NPM package? If they are, we should stick this in a typeRoots folder and use `declare module 'npm-check'`

Reply:
No other package that I could find across rushstack does this today, I think this would be a great optimization. I moved the typings to root of project for better visibility (which other projects do across rushstack).